pkg/initializer/initializer.go

```diff
@@ -0,0 +1,236 @@
+package initializer
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+)
+
+const (
+	DockerFileFmt = `FROM alpine:3.6
+
+ADD ./%s-e2e /e2e
+
+ENTRYPOINT ["/e2e"]
+`
+	MainGoContentFmt = `package main
```

Comment:
I really dislike having go content in a constant here. I would much prefer to have it in an actual file which is included. This has to be possible. The current state just removes all our editor goodness from the code.

Reply:
Yep agree, but don't know how to do it differently, the only thing that comes to mind is code generation, but imo it's too complex just for dropping these snippets. It might involve using templates for the files too, so that they wouldn't be proper go files either.

However we are looking at them from the producer project, once you initialize the project from the consumer project point of view you will get the go files ready to use, let me know wdyt.

Reply:
Mhm okay, I am not really into it that much but template files would be preferable to me. We can iterate on this though.

Reply:
The odd thing about separate template files in go is the way of building them into the compiled binary. There was a lot of hustle already where we tried different approaches. The less evil we found was having them in go code as strings.

Reply:
Yeah you're right. I don't like it but it seems like strings are the path of least resistance for now.